spec/unit/knife/core/windows_bootstrap_context_spec.rb

```diff
@@ -216,28 +225,26 @@
     end
   end
 
+  # TODO the 'through WinRM' and 'through SSH' tests below are exactly the same - what difference are they supposed
```

Comment:
I discovered that these two tests are exactly the same. Anyone know what they are supposed to be testing?